src/AzureDBExperiences.ts

```diff
@@ -93,7 +93,7 @@ export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience
 // Mongo is distinguished by having kind="MongoDB". All others have kind="GlobalDocumentDB"
 // Table and Gremlin are distinguished from SQL by their capabilities
 const CoreExperience: Experience = { api: API.Core, longName: "Core", description: "(SQL)", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB, tag: "Core (SQL)" };
-const MongoExperience: Experience = { api: API.MongoDB, longName: "Azure Cosmos DB for MongoDB API", shortName: "MongoDB", kind: DBAccountKind.MongoDB, tag: "Azure Cosmos DB for MongoDB API" };
+const MongoExperience: Experience = { api: API.MongoDB, longName: "MongoDB", shortName: "MongoDB", kind: DBAccountKind.MongoDB, tag: "MongoDB" };
```

Comment:
I think https://github.com/microsoft/vscode-cosmosdb/issues/1608 specifically refers to the prompt in the case of attaching a database account, not in all cases.

Also, Matt requested that we update the "Database Account API" placeholder text when attaching a database account.